### PR TITLE
feat: user facade api & onCommented  & cellPointerdown & hide selection

### PR DESCRIPTION
**Pull request by lumixraku**
close #xxx

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

**Review by weird94 — APPROVED** [review]

File: packages/core/src/facade/f-univer.ts [hunk 1]
```diff
@@ -162,4 +163,8 @@ export class FUniver extends FBase {
     newBlob(): FBlob {
         return this._injector.createInstance(FBlob);
     }
+
+    getUserManager(): FUser {
+        return this._injector.createInstance(FUser);
```
Comment (wzhudev) [inline packages/core/src/facade/f-univer.ts:168]: If the method's name is `getUserManager`, the Facade class should be `FUserManager` as well.

File: packages/core/src/facade/f-user.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { IUser } from '../services/user-manager/user-manager.service';
+import { Inject, Injector } from '../common/di';
+import { UserManagerService } from '../services/user-manager/user-manager.service';
+import { FBase } from './f-base';
+
+export class FUser extends FBase {
```
Comment (wzhudev) [inline packages/core/src/facade/f-user.ts:22]:
```suggestion
export class FUserManager extends FBase {
```

File: packages/core/src/facade/f-user.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        @Inject(UserManagerService) private readonly _userManagerService: UserManagerService
+    ) {
+        super();
+    }
+
+    /**
+     * Get current user info
+     * e.g. `univerAPI.getUserManager().getCurrUser()`
+     */
+    getCurrUser(): IUser {
```
Comment (wzhudev) [inline packages/core/src/facade/f-user.ts:34]:
```suggestion
    getCurrentUser(): IUser {
```

Do not use abbreviation in Facade API. Choose the mose direct way to instruct our users.

File: packages/sheets-thread-comment/src/facade/f-worksheet.ts [hunk 4]
```diff
@@ -32,6 +36,16 @@ export class FWorksheetCommentMixin extends FWorksheet implements IFWorksheetCom
         const comments = sheetsTheadCommentModel.getSubUnitAll(this._workbook.getUnitId(), this._worksheet.getSheetId());
         return comments.map((comment) => this._injector.createInstance(FThreadComment, comment));
     }
+
+    onCommented(callback: (params: IAddCommentCommandParams) => void): IDisposable {
```
Comment (wzhudev) [inline packages/sheets-thread-comment/src/facade/f-worksheet.ts:40]: JSDoc required.

File: packages/sheets-ui/src/facade/f-workbook.ts [hunk 5]
```diff
@@ -112,6 +112,20 @@ export class FWorkbookSheetsUIMixin extends FWorkbook implements IFWorkbookSheet
         );
     }
 
+    onCellPointerDown(callback: (cell: IHoverRichTextInfo) => void): IDisposable {
```
Comment (wzhudev) [inline packages/sheets-ui/src/facade/f-workbook.ts:115]: JSDoc required.

Comment (wzhudev) [reply packages/sheets-ui/src/facade/f-workbook.ts:115]: These method should also be defined in the interface.

**Review by wzhudev — CHANGES_REQUESTED** [review]

**Review by wzhudev — APPROVED** [review]
LGTM

**Pull request merged**